File: Python/BasicAlgorithms/PrintOddsTo255.py
#print odds 1-255

def PrintOddsTo255_For():
    for i in range(0,256):
        if i % 2 == 1:
            print  ("For to 255 value: ", i)

def PrintOddsTo255_While():
    i = 1
    while i < 256:
        if i % 2 == 1:
            print("While to 255 value: ", i)
        i += 1

File: Rust/BasicAlgorithms/src/PrintOddsTo255.rs
pub fn PrintOddsTo255_ForIn() {
    for i in 1..256 {
        if i % 2 == 1 {
            println!("{}", i);
        }
    }
}

pub fn PrintOddsTo255_While() {
    let mut i = 0;
    while i < 256 {
        if i % 2 == 1 {
            println!("{}", i);
        }
        i += 1;
    }
}